src/components/Dashboard/FirmsAddressList.vue

```diff
@@ -133,7 +133,11 @@ export default class FirmsAddressList extends Mixins(CommonMixin, CountriesProvi
   @Getter getBusinessAddress!: OfficeAddressIF
 
   // Business
-  get businessAddress (): OfficeAddressIF {
+  get businessAddress (): OfficeAddressIF|null {
+    let deliveryAddress = this.getBusinessAddress?.deliveryAddress
+    if (this.getBusinessAddress === null) return null
```

Comment:
'deliveryAddress' will be undefined if the [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) fails.

Also, these are good cases for just checking if [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) or [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy).

Reply:
Obsolete code. Resolving.